modules/_labs/pagination/react/lib/Pages.tsx

```diff
@@ -0,0 +1,98 @@
+/** @jsx jsx */
+import {css, jsx} from '@emotion/core';
+import type from '@workday/canvas-kit-labs-react-core';
+import {IconButton} from '@workday/canvas-kit-react-button';
+import canvas from '@workday/canvas-kit-react-core';
+import _ from 'lodash';
```

Comment:
```suggestion
import range from 'lodash/range';
```

This necessitates a change to the usage of lodash in this file as well.